src/main/java/uk/gov/beis/els/model/LegislationCategory.java

```diff
@@ -1,9 +1,12 @@
 package uk.gov.beis.els.model;
 
+import uk.gov.beis.els.categories.internetlabelling.model.InternetLabelColour;
```

Comment:
Ah I had that on before but Codacity then complained about unused imports because it had optimised something to a wildcard import: https://app.codacy.com/manual/jamesbarnett91/energy-label-service/commit?cid=502966400

Reply:
Do you have our code style loaded? https://confluence.fivium.co.uk/display/JAVA/Java+development+environment+setup#Javadevelopmentenvironmentsetup-Codestyle one of the rules in it should be to disable wildcard imports.

Reply:
Oh no, I had the Kotlin code style but not Java. Loaded and import optimisation enabled now